src/qt/sendcoinsdialog.cpp

```diff
@@ -263,7 +263,7 @@ bool SendCoinsDialog::PrepareSendText(QString& question_string, QString& informa
             {
                 recipients.append(entry->getValue());
             }
-            else if (valid)
+            else
```

Comment:
The `if (valid)` makes sure that `ensureWidgetVisible` is only called once, even if there are invalid entries. I'm not sure if this is important, either for performance or correctness, but it's something to consider.